src/main/java/pl/edu/pw/pik/pikactivitytrackerserver/controller/UserController.java

```diff
@@ -31,4 +32,30 @@
 
         return new ResponseEntity<>(null, HttpStatus.NOT_ACCEPTABLE);
     }
+
+    @GetMapping("/login/{username}")
+    public ResponseEntity<String> loginUsername(@PathVariable String username)
```

Comment:
Sugerowałbym zmianę nazwy funkcji, wg mnie jest trudno ją zrozumieć, ale to Ty zdecyduj ostatecznie

Reply:
zmieniłam na getSalt